js/main.js

```diff
@@ -0,0 +1,18 @@
+// main.js
+// Contains Javascript functions.....
+
+"use strict";
+
+(function() {
+  var soon = document.getElementsByClassName('soon')[0];
+
+  // Easter egg
+  document.getElementById("rawr").onclick = function() {
+      soon.innerHTML = "";
+      document.getElementsByClassName('banner')[0].style.display = "none";
+      document.getElementsByClassName('container')[0].style.paddingTop = 0;
+      var rawrburt = document.createElement("img")

```

Comment:
Whenever JavaScript declares variables it auto-magically hoists them to the top of the current scope (function)

So what happens is you get:
`var rawrburt;`
soon.innerHTML = "";
.
.
.
rawrburt = document.createElement("img")**;**
rawrburt.src = "images/rawrburt.png";

See this article for more information:
https://www.w3schools.com/js/js_hoisting.asp

Reply:
also `"strict mode";` should have enforced **;** to be used. I wonder why it didn't...